integration-tests/splunk-hec/pom.xml

```diff
@@ -51,9 +55,49 @@
             <artifactId>rest-assured</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.awaitility</groupId>
+            <artifactId>awaitility</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-integration-tests-support-splunk</artifactId>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+
     </dependencies>
 
     <profiles>
+        <profile>
```

Comment:
We'll need to add the `skip-testcontainers-tests` profile (see how its done elsewhere) because the Windows CI runners do not have Docker.